src-docs/src/components/with_theme/theme_context.tsx

```diff
@@ -17,7 +17,7 @@ import { applyTheme } from '../../services';
 const THEME_NAMES = OUI_THEMES.map(({ value }) => value);
 
 const defaultState = {
-  theme: THEME_NAMES[2],
+  theme: THEME_NAMES[0],
```

Comment:
while `0` is a safe and long-term value to use, i would rather have `OUI_THEMES` tell us which one is `default`.

Reply:
Will merge this to get doc site unblocked now, and implement this in a future PR